.github/workflows/create_release.yml

```diff
@@ -0,0 +1,90 @@
+name: Create Release
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  create_release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-client
+        path: downloads/ni-grpc-device-client
+
+    - uses: vimtor/action-zip@v1
+      with:
+        files: downloads/ni-grpc-device-client
+        dest: downloads/artifacts/ni-grpc-device-client.zip
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-client-Linux
+        path: downloads/artifacts
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-server-linux-glibc2_27-x64
+        path: downloads/artifacts
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-server-ni-linux-rt-x64
+        path: downloads/artifacts
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: ni-grpc-device-server-windows-x64
+        path: downloads/ni-grpc-device-server-windows-x64
+
+    - uses: vimtor/action-zip@v1
+      with:
+        files: downloads/ni-grpc-device-server-windows-x64
+        dest: downloads/artifacts/ni-grpc-device-server-windows-x64.zip
+
+    - name: "Get previous tag"
+      id: previoustag
+      run: >-
+        echo "::set-output name=tag::$(
+          git tag --list --sort=-version:refname | grep '^\(v[0-9]\+.[0-9]\+.[0-9]\+\)$' | head -1
```

Comment:
Note: I'll use this for the linux RT validation action too. `git tag` looks at all tags in the repo, not just those accessible from the current commit so it addresses that. Sorting by the reverse version should get the tags in latest release to first release order. The grep is to just get actual release tags... this avoids it returning any internal or rc suffixes we want to avoid.

Reply:
Does the regex match [rcs like this one](https://github.com/ni/grpc-device/releases/tag/v1.1.0-rc1)?

Reply:
No, it'll just get the tags "v*" with the three numbers separated by '.'s. I was thinking that this script would just assume you're making a normal release so it should get the last normal release tag (last "v*" tag without any hyphen suffix) to avoid any pre-releases.

Reply:
gotcha. that sounds reasonable. So, if you want to make `rc`s you need to change the name in the draft or make a new one or whatever. That sounds fine.